packages/core/src/components/Text.tsx

```diff
@@ -45,7 +45,7 @@ export const BaseLink = ({
   return (
     <Text
       hitSlop={8}
-      style={[{ color: theme.colors.primary, alignSelf: "flex-start" }, style]}
```

Comment:
This causes unexpected behavior when the parent of this component has `alignItems: "center"` set (P-2663)